src/OrchardCore.Modules/OrchardCore.Autoroute/GraphQL/AutorouteInputObjectType.cs

```diff
@@ -8,7 +8,7 @@ public class AutorouteInputObjectType : WhereInputObjectGraphType<AutoroutePart>
     {
         public AutorouteInputObjectType()
         {
-            Name = "AutoroutePartInput";
+            Name = "AutoroutePart";
```

Comment:
What's the reasoning for renaming all the input types? They are filters for the parts, so naming them as the actual part is confusing to me.

Reply:
You type in autoroutePart {} when you filter, and I now use this name instead of the part name to define what’s shown